common/reviews/api/heft-rspack-plugin.api.md

```diff
@@ -0,0 +1,67 @@
+## API Report File for "@rushstack/heft-rspack-plugin"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import type { AsyncParallelHook } from 'tapable';
+import type { AsyncSeriesBailHook } from 'tapable';
+import type { AsyncSeriesHook } from 'tapable';
+import type { AsyncSeriesWaterfallHook } from 'tapable';
+import type { HeftConfiguration } from '@rushstack/heft';
+import type { IHeftTaskSession } from '@rushstack/heft';
+import { rspackCore } from '@rspack/core';
+import type * as TRspack from '@rspack/core';
+import type * as TRspackDevServer from '@rspack/dev-server';
+
+// @beta (undocumented)
+export type IRspackConfiguration = TRspack.Configuration | TRspack.Configuration[];
+
+// @beta
+export interface IRspackConfigurationFnEnvironment {
+    heftConfiguration: HeftConfiguration;
+    prod: boolean;
+    production: boolean;
+    // Warning: (ae-forgotten-export) The symbol "RspackCoreImport" needs to be exported by the entry point index.d.ts
```

Comment:
Fix? Debatable since in practice the shape will be the shape of whichever version of rspack was loaded, so type-checking is a bit awkward.